android/java/res/layout/brave_toolbar.xml

```diff
@@ -25,7 +25,7 @@
             android:layout_marginBottom="8dp"
             android:src="@drawable/ic_picture_in_picture"
             android:layout_gravity="center"
-            android:contentDescription="@string/accessibility_toolbar_btn_brave_shields" />
+            android:contentDescription="@string/accessibility_toolbar_btn_brave_pip" />
```

Comment:
Sorry I just moved brave specific resources to a separate `brave-res` folder, you will need to rebase your PR